src/Panel/Panel.js

```diff
@@ -148,6 +149,7 @@ export default class Panel extends React.Component {
       header ?
         <PanelHeader
           title={header.hideTitleWhenExpanded && isExpanded ? undefined : header.title}
+          size={header.size || HeaderSize('small')}
```

Comment:
`small` should be moved to the `defaultProps` (so to have it documented in the README)